pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_uploader_tool.adoc

```diff
@@ -0,0 +1,33 @@
+[id="uploading-content-with-the-uploader-tool_{context}"]
```

Comment:
I think this discussion came up in another EUD PR, but ultimately, users will not have much use for the uploader tool until we implement sandboxes in Pantheon. At that point, users can follow this procedure to upload their docs to their personal sandbox.

I call this out because currently, this text as-written implies that that the uploader will be used to sync repository content, which is not the case.

Reply:
OK. I'll take a look. Maybe we need to just remove this topic.

Reply:
As per @adahms in another comment, we'll keep the module since it's already authored, but we don't have to include it when we publish the docs.